src/app/mork/api/v1/routers/users.py

```diff
@@ -164,6 +164,16 @@ async def update_user_status(
             status_code=status.HTTP_404_NOT_FOUND, detail=message
         ) from exc
 
+    try:
+        session.commit()
+    except SQLAlchemyError as exc:
+        session.rollback()
+        msg = f"Failed to update status of user {user_id}"
+        logger.error(msg)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail=msg
+        ) from exc
```

Comment:
Is this really needed ? If the exception bubble, is there an existing mechanism to rollback the transaction and then the exception is saved in sentry ?

Reply:
Indeed we can remove it!